crud/rooms.py

```diff
@@ -31,13 +31,13 @@ async def delete_room_type(id: int):
     return {"result": answer}
 
 
-async def get_rooms_types(skip: int = 0, limit: int = 100):
+async def get_room_types(skip: int = 0, limit: int = 100):
     """Get list of room's types"""
     results = await database.fetch_all(room_type_table.select().offset(skip).limit(limit))
```

Comment:
Executing all the CRUD operations based on id is good for now, but later you should add more parameters.